### ISSUE TITLE
SSL Not working :(

### ISSUE THREAD

**Issue by andrewyv** [AzuraCast/AzuraCast#2865]
The ssl is issued to localhost..

I don't understand why

nginx_proxy_letsencrypt_1  | 2020-05-21 19:40:15,044:INFO:simp_le:1450: Generating new certificate private key
nginx_proxy_letsencrypt_1  | 2020-05-21 19:40:20,226:ERROR:simp_le:1417: CA marked some of the authorizations as invalid, which likely means it could not access http://example.com/.well-known/acme-challenge/X. Did you set correct path in -d example.com:path or --default_root? Are all your domains accessible from the internet? Please check your domains' DNS entries, your host's network/firewall setup and your webserver config. If a domain's DNS entry has both A and AAAA fields set up, some CAs such as Let's Encrypt will perform the challenge validation over IPv6. If your DNS provider does not answer correctly to CAA records request, Let's Encrypt won't issue a certificate for your domain (see https://letsencrypt.org/docs/caa/). Failing authorizations: https://acme-v02.api.letsencrypt.org/acme/authz-v3/47284

**PS: i think i know why - my port for azura web is 8080 not default 80**

**Comment by SlvrEagle23** [comment]
@andrewyv Correct...you can run

```
./docker.sh change-ports
```

...to switch your ports to 80 and 443, then do the LetsEncrypt setup, and once it's been set up you can switch back. You will have to do this at least once every 90 days to fix the issue.

**Comment by andrewyv** [comment]
@SlvrEagle23  please can you make a fix for future release for LetsEncrypt to work without switching ports? thank you...
I would use on port 80 as default, but I need to host azuracast and my radio website on a powerful VM, because it's cheaper than having 2 VM with lower specs.

**Comment by Xara** [comment]
you can use a subdomain and  doing reverse proxy, as I do, Azura is served on 80 on 8080 and on station port, on 80 is  something.your_domain.tld., on 8080 is otherthing.your-domain.tld., then standard radio.domain.tld:port/mount
ah yes and also with Azura proxies it work as well.

**Comment by Ficreet** [comment]
I'm just commenting to let you know that I had the same problem as @andrewyv . This one was fixed after executing the provided command.
For my part, the connection ports to Azuracast are the basic ones. I did not modify them.
I hope that this problem does not occur again,

Thanks to you!

**Comment by siderissym** [comment]
Hello there.

I have the same problem. The ssl is issued to localhost although my ports are the defaults (80, 443, 2022)

Specifically, I have installed AzuraCast in other ports when I first installed it, then I changed them to default and tried `./docker.sh letsencrypt-create` without problems but the certificate still remains as localhost.
I also have restarted my VPS.

Any thoughts?

Thank you in advance
